modules/ROOT/pages/shared-resources.adoc

```diff
@@ -181,6 +198,7 @@ In Studio's visual editor, you can simply pick the shared resource out of the dr
 
 image::pick-resource.png[pick+resource]
 
+[[deploy-shared-resources]]
 == Deploying with Shared Resources
 
 In Anypoint Studio, when you deploy an application that is associated to a domain, by default Studio deploys both the application and the domain together. Also, when deploying a domain project, by default Studio deploys every application associated to it, as well. You can change these default behaviors by changing the Run Configuration for the domain. In fact, you can deploy any set of applications in your workspace together, even if they don't share the same domain.
```

Comment:
```suggestion
By default, when you deploy either an application associated with a domain or a domain associated with any application, Anypoint Studio deploys both. You can change this default behavior by changing the run configuration for the domain. You can also deploy a set of applications in your workspace together, even if they don't share the same domain.
```
Take another look at this para and decide if the final sentence really belongs there. It seems like a different behavior, unless changing the run config enables this as well...?